languages/brazilian/pack/src/main/res/xml/brazilian_keyboards.xml

```diff
@@ -11,4 +11,6 @@
 	    id="4c99178a-9a80-4218-8b13-f05f474c7f5e" defaultDictionaryLocale="pt_BR" description="Teclado QWERTY com Ç" index="111"/>
         <Keyboard nameResId="@string/brazilian_keyboard_name" iconResId="@drawable/ic_status_brazilian" layoutResId="@xml/brazilian_qwerty_with_symbols"
 	    id="848ac210-4d87-11e8-b566-0800200c9a66" defaultDictionaryLocale="pt_BR" description="Teclado QWERTY com símbolos" index="112"/>
+		<Keyboard nameResId="@string/brazilian_keyboard_name" iconResId="@drawable/ic_status_brazilian" layoutResId="@xml/qwerty"

```

Comment:
I had to remove this, it does not compile because of missing resources in `qwerty.xml`

Reply:
I'm confused with these resources, I tried to make like the other keyboards that are there